script/download_rust_deps.py

```diff
@@ -49,7 +50,7 @@ def DownloadAndUnpackRustDeps(platform):
     try:
         deps.DownloadAndUnpack(url, RUSTUP_DIR)
```

Comment:
DownloadAndUnpack will still create the rustup dir if it is missing however the S3 dep packages now unzip to a version folder so that the checks are reliable